### DISCUSSION TITLE
Это так задумано - непонятно как запустить программу?

### DISCUSSION THREAD

**Discussion by tsvigo** [lllyasviel/Fooocus#1907] [Q&A]
Программа великолепная. Но непонятно как её запустить — каждый раз танцы с бубном и проход по всем трём способам запустить на Линуксе причём идёт скачивание каждый раз модели на 7 гигабайтов.

**Comment by mashb1t** [comment]
Please make sure to only use one of the three installation methods, not all three together.

Either
https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#linux-using-anaconda
or
https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#linux-using-python-venv
or
https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#linux-using-native-system-python
